test/Prompt/ValuePrinter/Regression.C

```diff
@@ -56,8 +56,14 @@ nullptr // CHECK: (nullptr_t) nullptr
 true // CHECK: (bool) true
 false // CHECK: (bool) false
 
+unordered_multiset<float> {1} // ROOT-7310
+// expected-error@2 {{use of undeclared identifier 'unordered_multiset'}}
```

Comment:
...but we are not running verify on this - who should consume the `expected-error`s?